### ISSUE TITLE
New users insight

### ISSUE THREAD

**Issue by KaiOelfke** [TelemetryDeck/Viewer#307]
I think it would be good to not only have daily active users in the overview, but also new users / installs / downloads. This is probably possible with custom queries, but I'm not sure how.

**Comment by winsmith** [comment]
I agree! This fits with my plan to have more pre-packaged insights in general! :)

**Comment by alexstaravoitau** [comment]
@winsmith Is it possible to create a new users / installs insight now, with a custom query for example?

**Comment by winsmith** [comment]
I don't think so because I can't think of a criterion that makes a user a new user that doesn't need me to look at the entirety of the data set for the possibility of the user having previously existing. 

I think the way to go is to store a value on the client side. Something like "SDK has been initialized before" stored in userdefaults. When that value is missing, send a "first time user" signal.

**Comment by alexstaravoitau** [comment]
Thanks @winsmith — yep that will work, I was just hoping I could do this without releasing an update.

**Comment by keleftheriou** [comment]
I want to add my vote for this feature -- tracking new users is a very common need, and feels like it belongs inside TelemetryDeck rather than needing to be implemented by each client separately.
